var/spack/repos/builtin/packages/pmlib/fix_compiler_options.patch

```diff
@@ -23,7 +23,7 @@ diff -urN PMlib-org/cmake/Toolchain_fx100.cmake PMlib-fix/cmake/Toolchain_fx100.
 -  CMAKE_FORCE_CXX_COMPILER(mpiFCCpx GNU)
 -  CMAKE_FORCE_Fortran_COMPILER(mpifrtpx GNU)
 +  CMAKE_FORCE_C_COMPILER(mpicc GNU)
-+  CMAKE_FORCE_CXX_COMPILER(mpicc GNU)
++  CMAKE_FORCE_CXX_COMPILER(mpicxx GNU)
```

Comment:
Sorry for the lack of confirmation. 
There was no need to correct this problem in my environment.
But, I correct this part to be more flexibility.